### PR TITLE
Refactored filters added sort.

### PR DESCRIPTION
**Pull request by kaustubh-jsr**
Files to check
ProductListingContainer.jsx
Filters.js
FilterSubcatInput.jsx
filterReducer.jsx

### CODE REVIEW

**Review by anasm11 — APPROVED** [review]
The code looks clean. You can merge.

File: src/components/Cart/EmptyCartSvg.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./EmptyCartSvg.css";
+function EmptyCartSvg() {
+  return (
+    <div className="empty-cart">
+      <img
+        src="https://i.postimg.cc/KzgCFYHz/empty-cart-2130356-1800917.png"
+        alt=""
```
Comment (Naman-Saxena1) [inline src/components/Cart/EmptyCartSvg.jsx:8]: alt is empty

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks fine, was not able to find any particular issues
For UI I feel it would look better if you shift Fruits and Vegetable heading in the center and increase its font size

## Before
Capture4
 

## After
Capture5

File: src/components/Cart/EmptyCartSvg.jsx (hunk 1 above)
Comment (kaustubh-jsr) [reply src/components/Cart/EmptyCartSvg.jsx:8]: Fixed that, thanks for pointing out.

**Pull request merged**